middleware/feed.ts

```diff
@@ -0,0 +1,8 @@
+export default defineNuxtRouteMiddleware((from) => {
+  if (!from.params.feed || !validFeeds.includes(from.params.feed as string)) {
```

Comment:
Where is defined validFeeds?

I will redirect directly to validFeeds[0] + '/1' to avoid multiple redirects

Reply:
https://github.com/nuxt/hackernews/blob/e160b93f1f1be4825f9690936b8b948e618c4f1e/composables/api.ts#L9

and that's a good point!